docs/buildah-from.md

```diff
@@ -126,6 +138,16 @@ The [username[:password]] to use to authenticate with the registry if required.
 If one or both values are not supplied, a command line prompt will appear and the
 value can be entered.  The password is entered without echo.
 
+**--ipc** *how*
+
+Sets the configuration for IPC namespaces when the container is subsequently
+used for `buildah run`.
+The configured value can be "" (the empty string) or "container" to indicate
```

Comment:
How would they specify container?   Via the id and/or name?  Ditto host, short name, fqdn, ip?  This verbiage is probably OK as is, but examples would be greatly appreciated.